charts/eks-pod-identity-agent/values.yaml

```diff
@@ -79,4 +79,13 @@ tolerations:
 updateStrategy:
   type: RollingUpdate
   rollingUpdate:
-    maxUnavailable: "10%"
\ No newline at end of file
+    maxUnavailable: "10%"
+
+# This setting can be used to create service account and enable IRSA
+# instead of relying on EC2 instance credentials.
+# for enhanced security recommended to use together with AWS_EC2_METADATA_DISABLED=true
+irsa:
+  enabled: false
+  service_account:
+    role_arn: IRSA_ROLE_ARN
+    additionalAnnotations:
```

Comment:
Wondering if an empty YAML could key be an issue for the tooling (Helm/Flux/linter)?
Should we use `additionalAnnotations: {}` instead?

Reply:
I modelled it after:
https://github.com/gravitational/eks-pod-identity-agent/blob/ca897a886c64bb8985e64ee2ffd64e2eb3e57104/charts/eks-pod-identity-agent/values.yaml#L24-L26


Also tested with `helm template` command, so should be fine